docker-compose.yml

```diff
@@ -4,22 +4,26 @@ services:
     image: nginx:1.14-alpine
     expose:
       - "80"
-    ports: 
+    ports:
       - "80:80"
     volumes:
       - ./conf/nginx.conf:/etc/nginx/nginx.conf:ro
     links:
       - meeds
   meeds:
     image: meeds-io/meeds
+    expose:
+      - "8080"
+    ports:
+      - "8080:8080"
```

Comment:
In fact, we gave the possibility to change 8080 port mapping using command line as described in README:

> docker run -v meeds_data:/srv/meeds -p 8080:8080 meeds-io/meeds

And we exposed 8080 already in

https://github.com/Meeds-io/meeds-docker/blob/develop/Dockerfile#L96

Reply:
I think it would be better to have exposed default when

Reply:
I think it would be better to have this port exposed by default in `docker-compose` mode, would be useful in case vhost is not accessible. I have to deployed it in a VM then tried to access via IP and got redirect to `localhost`.

Reply:
Hi @uocnb 
Regarding the ``expose`` section, I agree with @boubaker, the expose is redundant with the expose already declared in the dockerfile.
Regarding the ``ports`` section, I would not encourage to expose by default the tomcat to the worldwide. The first point is for security reasons as the request will not be logged at all (the tomcat access logs are not activated by default), but also you will lost the advantage of having a nginx as entrypoint (custom rules, ssl termination, http/2, caching, ...).

The docker-compose file is just provided for test purposes and needs to be adapted for "production ready" or more industrialized deployments. You can do it by extending the configuration with a [``docker-compose-override.yml`` file](https://docs.docker.com/compose/extends/) or write your own docker-compose.yml file.

For your particular problem of the redirection to ``locahost``, the ``MEEDS_PROXY_VHOST`` needs to be changed to your domain name or your vm's ips
If you are considering more than a simple test, I also encourage you to deploy with a real database container (postgresql or mysql).

Reply:
Hi @vsellier,
I agree with your security point of view, will revert it.

Reply:
I've updated this PR! please check again, thank you!